src/tests/system/tests/test_ipa_trusts.py

```diff
@@ -60,3 +62,130 @@ def test_ipa_trusts__lookup_group_without_sid(ipa: IPA, trusted: GenericADProvid
     assert "online status: offline" not in status.stdout.lower(), "AD domain went offline!"
     assert "online status: online" in status.stdout.lower(), "AD domain was not online!"
+
+
+@pytest.mark.importance("medium")
+@pytest.mark.ticket(jira="RHEL-14752")
+@pytest.mark.topology(KnownTopology.IPATrustIPA)
+def test_ipa_trusts__ipa_master_lookup_trusted_user_and_group(ipa: IPA, trusted: IPA):
+    """
+    :title: Basic IPA-IPA Trust lookup on IPA server
+    :setup:
+        1. Restart SSSD and clear cache on IPA server
+    :steps:
+        1. Resolve trusted domain admin user
+        2. Resolve trusted domain admins group
+    :expectedresults:
+        1. User is resolved
+        2. Group is resolved
+    :customerscenario: True
+    """
+    ipa.sssd.restart(clean=True)
+
+    admin = trusted.admin_fqn
+    id_user = ipa.tools.id(admin)
+
+    admins = trusted.fqn("admins")
+    getent_group = ipa.tools.getent.group(admins)
+
+    assert id_user is not None, "Trusted admin user not found!"
+    assert id_user.user.name == admin, "Username does not match!"
+
+    assert getent_group is not None, f"No group {admins} found!"
+    assert getent_group.name == admins, f"Group name does not match {admins}!"
+
+
+@pytest.mark.importance("medium")
+@pytest.mark.ticket(jira="RHEL-14752")
+@pytest.mark.topology(KnownTopology.IPATrustIPA)
+def test_ipa_trusts__lookup_trusted_user_and_group(client: Client, ipa: IPA, trusted: IPA):
+    """
+    :title: Basic IPA-IPA Trust lookup on IPA client
+    :setup:
+        1. Restart SSSD and clear cache on IPA client
+    :steps:
+        1. Resolve trusted admin user
+        2. Resolve group "admins@trusteddomain"
+    :expectedresults:
+        1. User is resolved
+        2. Group is resolved
+    :customerscenario: True
+    """
+    client.sssd.restart(clean=True)
+
+    admin = trusted.admin_fqn
+    id_user = client.tools.id(admin)
+
+    admins = trusted.fqn("admins")
+    getent_group = client.tools.getent.group(admins)
+
+    assert id_user is not None, "Trusted admin user not found!"
+    assert id_user.user.name == admin, "Username does not match!"
+
+    assert getent_group is not None, f"No group {admins} found!"
+    assert getent_group.name == admins, f"Group name does not match {admins}!"
+
+
+@pytest.mark.importance("critical")
+@pytest.mark.topology(KnownTopologyGroup.AnyIPATrust)
+def test_ipa_trusts__user_login(client: Client, trusted: GenericProvider):
+    """
+    :title: Authenticate IPA and trusted AD users with default settings
+    :setup:
+        1. Create trusted user
+        2. Start SSSD
+    :steps:
+        1. Authenticate user using their fully qualified name
+        2. Authenticate user using the wrong password
+    :expectedresults:
+        1. Login is successful
+        2. Login is unsuccessful
+    :customerscenario: False
+    """
+    trusted_user = trusted.user("user1").add(password="Secret123").name
```

Comment:
nitpick, all the tests so far, only test a trusted user. I think we should just use ```user1```, ```user1_fqn``` for the variable names.

Reply:
Okay I changed `trusted_user` variable name to `user1`  and `trusted_user_fqn` changed to `user1_fqn` in both tests.